build.sc

```diff
@@ -560,7 +561,8 @@ object main extends MillStableScalaModule with BuildInfo {
     Deps.windowsAnsi,
     Deps.mainargs,
     Deps.coursierInterface,
-    Deps.requests
+    Deps.requests,
+    Deps.sonatypeCentralClient
```

Comment:
Nitpick: Maybe add a trailing comma here, so the next time someone needs to add a line, it doesn't change an unrelated line (like `Deps.requests` here).

Reply:
That would not be in line with our current formatting rules

Reply:
This was removed anyway in the process of moving everything to a contrib module - resolving.